server/src/main/java/com/unosquare/carmigo/service/JourneyService.java

```diff
@@ -1,29 +1,85 @@
 package com.unosquare.carmigo.service;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.fge.jsonpatch.JsonPatch;
+import com.github.fge.jsonpatch.JsonPatchException;
+import com.unosquare.carmigo.dto.CreateJourneyDTO;
+import com.unosquare.carmigo.dto.GrabJourneyDTO;
+import com.unosquare.carmigo.entity.Driver;
+import com.unosquare.carmigo.entity.Journey;
+import com.unosquare.carmigo.entity.Location;
+import com.unosquare.carmigo.exception.ResourceNotFoundException;
+import com.unosquare.carmigo.repository.JourneyRepository;
+import com.unosquare.carmigo.util.MapperUtils;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
 import org.springframework.stereotype.Service;
 
-import java.util.Map;
+import javax.persistence.EntityManager;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
 
 @Service
-public class JourneyService {
+@RequiredArgsConstructor
+public class JourneyService
+{
+    private final JourneyRepository journeyRepository;
+    private final ModelMapper modelMapper;
+    private final ObjectMapper objectMapper;
+    private final EntityManager entityManager;
 
-    public static String getJourneyParameters(final Map<String, String> paramMap)
+    public GrabJourneyDTO getJourneyById(final int id)
     {
-        if (!paramMap.isEmpty()) {
```

Comment:
Instead of most of the logic of the method in an if statement, perhaps consider the following at the start of the method.  It reads better :)
`
if (paramMap.isEmpty()} { return; }`